docs/source/export.md

```diff
@@ -0,0 +1,189 @@
+Export
+=====
+
+1. [Introduction](#introduction)
+
+2. [Supported Framework Model Matrix](#supported-framework-model-matrix)
+
+3. [Examples](#examples)
+
+4. [Appendix](#appendix)
+
+# Introduction
+Open Neural Network eXchange (ONNX) is an open standard format for representing machine learning models. Exporting FP32 PyTorch/Tensorflow models has become popular and easy to use. However, for Intel Neural Compressor, we hope to export the int8 model to ONNX to achieve higher applicability on multiple frameworks.
+
+Here we briefly introduce our export API for PyTorch FP32/INT8 models. First, the int8 ONNX model is not directly exported from the int8 PyTorch model, but quantized after obtaining the FP32 ONNX model using the mature torch.onnx.export API. To ensure that the quantization process of ONNX is as consistent as possible with PyTorch, we reuse three key pieces of information from the Neural Compressor model to perform ONNX quantization.
```

Comment:
should be "is consistent with"

Reply:
Actually, they are not consistent in all patterns.